### PR TITLE
fix: add kartikver15gr8

### LINKED ISSUE #1355: Kartikey Verma
## Profile Screenshot

Screenshot 2022-09-03 161410

### PR DESCRIPTION
**Pull request by kartikver15gr8**
Added my profile in the profile folder of awesome-github-profiles

resolves #1355 
## Checklist for profile addition

- [x] Name the PR `fix: add kartikver15gr8`.
- [x] You've added file named `your-kartikver15gr8.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/kartikver15gr8.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/91731654/188267011-44c4fd62-260f-4b8c-b413-ed48916176d0.png",
+    "issueId": 1355,
+    "name": "Kartikey Verma",
+    "username": "kartikver15gr8"
+  }
```
Comment (kaiwalyakoparkar) [inline profiles/kartikver15gr8.json:6]: Please remove the extra space before the closing bracket
```diff
-  }
+ }
```

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: profiles/kartikver15gr8.json (hunk 1 above)
Comment (kartikver15gr8) [reply profiles/kartikver15gr8.json:6]: Sure

**Review by kartikver15gr8 — COMMENTED** [review]
sure

**Comment by kartikver15gr8** [comment]
I've removed extra spaces before the closing curly braces

**Comment by kartikver15gr8** [comment]
I have removed those extra spaces, Kindly merge it into main 😊

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, thank you for the PR. Just need to fix the minor indentation issue.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

File: profiles/kartikver15gr8.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/91731654/188267011-44c4fd62-260f-4b8c-b413-ed48916176d0.png",
+    "issueId": 1355,
+    "name": "Kartikey Verma",
+    "username": "kartikver15gr8"
+}
```
Comment (Pradumnasaraf) [inline profiles/kartikver15gr8.json:6]:
```suggestion
{
  "image": "https://user-images.githubusercontent.com/91731654/188267011-44c4fd62-260f-4b8c-b413-ed48916176d0.png",
  "issueId": 1355,
  "name": "Kartikey Verma",
  "username": "kartikver15gr8"
}

```

**Comment by kartikver15gr8** [comment]
Please check now.

**Pull request merged**